wgpu-types/src/lib.rs

```diff
@@ -593,6 +607,32 @@ impl Default for Limits {
     }
 }
 
+impl Limits {
+    /// These default limits are guarenteed to be compatible with GLES3, WebGL, and D3D11
+    pub fn downlevel_limits() -> Self {
```

Comment:
can we rename this method a bit? `Limits::downlevel_limits` is a bit repeat-y.
Maybe `Limits::downlevel_default` or something like this?

Reply:
Amusingly that's the name I put in the PR description, will change :)